packages/react-native-batch/src/__tests__/withReactNativeBatchAppBuildGradle-test.ts

```diff
@@ -0,0 +1,13 @@
+import { pushDependencies } from "../withReactNativeBatchAppBuildGradle";
+import {
+  buildGradleFixture,
+  buildGradleExpectedFixture,
+} from "./fixtures/buildGradle";
+
+describe("pushDependencies", () => {
```

Comment:
```suggestion
describe(pushDependencies, () => {
```

Reply:
do same for all tests of specific functions

Reply:
ok, I didn't know I can give a function as parameter